src/azure-cli/azure/cli/command_modules/containerapp/custom.py

```diff
@@ -1223,8 +1223,17 @@ def update_containerappsjob_logic(cmd,
                     scale_rule_def["auth"] = auth_def
                     if not scale_def:
                         scale_def = JobScaleModel
-                    scale_def["rules"] = [scale_rule_def]
-                    eventTriggerConfig_def["scale"]["rules"] = scale_def["rules"]
+                    if "rules" not in eventTriggerConfig_def["scale"]:
```

Comment:
Recommend using `safe_get` to avoid `NoneType` error.

Reply:
@Greedygre, I have updated this to use `safe_get`